src/main/factories/controllers/forget-password-factory.ts

```diff
@@ -10,11 +10,11 @@ import env from '@/main/config/env'
 
 export const makeForgetPasswordController = (): Controller => {
   const emailValidator = new EmailValidatorAdapter()
-  const loadGuardianByEmailRepository = new GuardianAccountRepository()
+  const guardianRepository = new GuardianAccountRepository()
   const salt = Number(env.salt)
   const bcryptAdapter = new BcryptAdapter(salt)
   const saveTokenRepository = new GuardianAccountRepository()
-  const tokenGenerator = new ForgetPasswordTokenGenerator(bcryptAdapter, saveTokenRepository)
+  const tokenService = new ForgetPasswordTokenGenerator(bcryptAdapter, saveTokenRepository)
```

Comment:
With the move to `guardianRepository` instead of `loadGuardianByEmailRepository` and others, shouldn't `ForgetPasswordTokenGenerator` also get a `guardianService` as a dependency instead of `saveTokenRepository`?

Reply:
Yes, I will rename it.